scripts/generate_graph.py

```
@@ -0,0 +1,117 @@
+
+import tempfile
+tmppath = tempfile.mkdtemp()
+
+#### increase the number of scans generated to 3 if running group analysis
+generate_token_data(tmppath, number_scans=3)
+
+output_folder = f'{tmppath}/outputs'
+
+#### HERE ARE SET THE DESIRED PARAMETERS FOR PREPROCESSING
+args = [
+        f'--exclusion_ids',f'{tmppath}/inputs/sub-token1_bold.nii.gz',
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)